controllers/users.js

```diff
@@ -292,6 +292,19 @@ const rejectProfileDiff = async (req, res) => {
   }
 };
 
+const addUserIntro = async (req, res) => {
+  await userQuery.addJoinData(req.body);
+  return res.json({ message: "User Data Added Succesfully" });
+};
+
+const getUserIntro = async (req, res) => {
+  const data = await userQuery.getJoinData(req.params.username);
+  return res.json({
+    message: "Got user data",
+    data: data,
+  });
+};
```

Comment:
put this also in a try catch block, and the message can be changed to something like `user data returned`